sklearn/tests/test_multiclass.py

```diff
@@ -926,3 +927,70 @@ def test_ovo_consistent_binary_classification():
     ovo.fit(X, y)
 
     assert_array_equal(clf.predict(X), ovo.predict(X))
+
+
+# Test that metadata is routed correctly for multiclass-classifiers
+# =================================================================
+
+
+@pytest.mark.usefixtures("enable_slep006")
+def test_fitted_coefs_differ_when_sample_weight_differs():
+    """Assert if coefficients differ when estimators are fitted with different
```

Comment:
```suggestion
    """Assert that coefficients differ when estimators are fitted with different
```

using "if" here does not inform me explicitely if they should be different or not :)